README.md

```diff
@@ -46,4 +46,4 @@ const rejectedPromise = Promise.reject('some reason');
     console.log(await isPromiseNotRejected(resolvedPromise));
     // true
 })();
-```
\ No newline at end of file
+```
```

Comment:
I'm not sure what this diff is, I just used to GitHub online editor for the typo fix. Perhaps a line ending change or something?